frontend/src/components/podcast/helpers/get-columns.ts

```diff
@@ -0,0 +1,21 @@
+import { Column } from 'react-table';
+
+const getColumns = (columnsScheme: Record<string, string>): Column[] => {
+  const columns: Column[] = [
+    {
+      Header: '#',
+      accessor: (_originalRow, rowIndex): string => String(rowIndex),
+    },
+  ];
+
+  Object.keys(columnsScheme).forEach((key) => {
+    columns.push({
+      Header: columnsScheme[key],
+      accessor: key,
+    });
+  });
```

Comment:
```suggestion
  const columns: Column[] = [
    {
      Header: '#',
      accessor: (_originalRow, rowIndex): string => String(rowIndex),
    },
    {
      // column
    }
  ];

  Object.keys(columnsScheme).forEach((key) => {
    columns.push({
      Header: columnsScheme[key],
      accessor: key,
    });
  });
```
describe all columns manually